ChangeLog

```diff
@@ -61,6 +61,11 @@ Features
    * Add MBEDTLS_REMOVE_3DES_CIPHERSUITES to allow removing 3DES ciphersuites
      from the default list (enabled by default). See
      https://sweet32.info/SWEET32_CCS16.pdf.
+   * Add support for parsing otherName entries in the Subject Alternative Name
```

Comment:
Minor: RFC 4108 Section 5 is specifically about hardware module name, which this ChangeLog entry doesn't entail.